docs/processes/documentation/documentation.md

```diff
@@ -0,0 +1,26 @@
+## Introduction
```

Comment:
@talzid @sksingh2022 What is the difference between this doc and the [one](https://github.com/thoth-tech/handbook/blob//docs/documentation/index.md) we current have in this folder?